.github/workflows/static-analysis.yml

```diff
@@ -2,7 +2,6 @@
 
 on:
     - pull_request
-    - push
```

Comment:
@drupol do you want these to run in `master` as well, after a merge?

Currently we have them set up like that but I don't know if it was intentional or not: https://github.com/loophp/collection/actions?query=branch%3Amaster

Reply:
I don't think it's necessary. All the changes happening in this project should be committed through PRs and not directly in the master branch, so... it will be tested in PRs. WDYT ?

Reply:
![Screenshot 2021-07-07 at 09 22 07](https://user-images.githubusercontent.com/34811569/124725493-d1f09400-df04-11eb-88d3-5b8aefa1b14e.png)

I think some of the badges on the readme depend on that 🤔

Reply:
Fair enough, let's not modify these.